### ISSUE TITLE
Why?

### ISSUE THREAD

**Issue by dovy** [GoogleCloudPlatform/appengine-symfony-starter-project#5]
This project keeps getting abandoned. I really want to use app-engine.  :(

**Comment by bshaffer** [comment]
This project isn't abandoned!

**Comment by dovy** [comment]
Well it doesn't work at all. :(

**Comment by JustinBeckwith** [comment]
@dovy can you be more specific?
- What exactly doesn't work.  Does the app run locally?  Can you deploy it?  Are you getting an error message?
- What steps are you taking to end up in this state?

The more details you can give us, the better chance we have of fixing what's broken.

**Comment by dovy** [comment]
`$ composer create-project google/appengine-symfony-starter-project`

But just got:

> [InvalidArgumentException] Could not find package google/appengine-symfony-starter-project with stability stable.

So instead I just cloned the repo:

As instructed, modified app.yaml:

```
env_variables:
  GCS_BUCKET_NAME: "website-cache"
```

Also modified php.ini:

```
google_app_engine.allow_include_gs_buckets=website-cache
```

Attempted to deploy. Getting this:

> Warning: require(/base/data/home/apps/s~flash-student-96619/20160219t113532.390817043465362270/app/../vendor/autoload.php): failed to open stream: No such file or directory in /base/data/home/apps/s~flash-student-96619/20160219t113532.390817043465362270/app/autoload.php on line 18 Fatal error: require(): Failed opening required '/base/data/home/apps/s~flash-student-96619/20160219t113532.390817043465362270/app/../vendor/autoload.php' (include_path='.;/base/data/home/apps/s~flash-student-96619/20160219t113532.390817043465362270/;/base/data/home/runtimes/php/sdk') in /base/data/home/apps/s~flash-student-96619/20160219t113532.390817043465362270/app/autoload.php on line 18

So tried using a different bucket name, full google URL:
`gs://flash-student-96619/website-cache`

> Warning: require_once(/base/data/home/apps/s~flash-student-96619/20160219t113714.390817071922918964/web/../app/bootstrap.php.cache): failed to open stream: No such file or directory in /base/data/home/apps/s~flash-student-96619/20160219t113714.390817071922918964/web/clear_cache.php on line 21 Fatal error: require_once(): Failed opening required '/base/data/home/apps/s~flash-student-96619/20160219t113714.390817071922918964/web/../app/bootstrap.php.cache' (include_path='.;/base/data/home/apps/s~flash-student-96619/20160219t113714.390817071922918964/;/base/data/home/runtimes/php/sdk') in /base/data/home/apps/s~flash-student-96619/20160219t113714.390817071922918964/web/clear_cache.php on line 21

That didn't work, so tried the following, but with the same results:

> flash-student-96619/website-cache
> gs://website-cache

:(

The reason I thought it was dead was it's been a while since the last commit/issue. Either way, if you can direct me, I have a backend app I'd love to fully move to appengine. Hope I've cleared out enough files for it to post.

Works perfectly locally BTW.  ;)

**Comment by bshaffer** [comment]
`Failed opening required '../vendor/autoload.php'`

You ran `composer install` in the project root?

**Comment by dovy** [comment]
Duh, haha.

Ok, now progressed to:

> Warning: fopen(gs://website-cache/symfony/cache/profiler/index.csv): failed to open stream: "\google\appengine\ext\cloud_storage_streams\CloudStorageStreamWrapper::stream_open" call failed in /base/data/home/apps/s~flash-student-96619/20160219t163825.390821932371969206/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Profiler/FileProfilerStorage.php on line 160

**Comment by bshaffer** [comment]
hmm, I'm not sure about that one. Is the warning causing the whole app to fail? If so, try clearing the cache.

**Comment by bshaffer** [comment]
@dovy you're right! The first time I deploy, it works great. But upon re-deploying, it blows up because the previously referenced files are no longer accessible.

I believe I need to change some reference to use relative paths instead of absolute ones.

**Comment by dovy** [comment]
Whew. So glad I'm not the only one.  :)

Let me know when this is resolved and I'll test it again. Thanks!

**Comment by bshaffer** [comment]
fixed in #7
